eng/CreateVersionFile.csproj

```diff
@@ -75,7 +75,7 @@
     <PropertyGroup>
       <NativeVersionLines>
 <![CDATA[
-static char sccsid[] __attribute__((used)) = "@(#)Version $(InformationalVersion)$(BuiltByString)";
```

Comment:
Any reason to remove the static? It's what we use elsewhere https://github.com/dotnet/arcade/blob/7a2f81b46ec807fe5df62f6df9c26bd3daee4fcf/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.targets#L118

Reply:
I'm using it to print out the SOS version on xplat in the eeversion command and "static" prevent it from being seen/extern'ed.

Reply:
It looks like we still use the version.cpp generated by our CreateVersionFile.csproj instead of arcade's unless both are linked?

Reply:
We added this before they exposed the target and then we never moved. I just don't know why it's not seen. I always use that variable to ask for a runtime version. Although lately I've noticed it seems repeated.